backend/api/src/main/kotlin/software/shonk/application/service/ShorkService.kt

```diff
@@ -70,6 +71,33 @@ class ShorkService(private val shork: IShork) : ShorkUseCase {
         return Result.success(lobby.getStatus())
     }
 
+    override fun getAllLobbies(): AllLobbies {
+        val lobbiesStatus = mutableListOf<Result<Status>>()
+        var lobbyId = 0L
+        while (true) {
+            val currentLobbyStatus = getLobbyStatus(lobbyId)
+            if (currentLobbyStatus.getOrNull() == null) {
+                break
+            }
+            lobbiesStatus.add(currentLobbyStatus)
+            lobbyId++
+        }
```

Comment:
This will break the second there is a gap in the lobby IDs.
Suppose we have the lobbies with IDs 0, 1, 3, 4
Lobbies 3 and 4 would never get looked at.

Reply:
I found the solution to this problem, I used lobbies map to call and check all its keys and then store those in a 2 lists; one for Lobbies' Status and one for IDs. Both add new values at the same time so there's no room for error in the index